address.md

```diff
@@ -0,0 +1,300 @@
+# Address
+
+A Filecoin address is an identifier that refers to an actor in the Filecoin state. All actors (miner actors, the storage market actor, account actors) have an address. This address encodes information about the network to which an actor belongs, the specific type of address encoding, the address payload itself, and a checksum. The goal of this format is to provide a robust address format that is both easy to use and resistant to errors.
+
+#### Design criteria
+
+1. **Identifiable**: The address must be easily identifiable as a Filecoin address.
+2. **Reliable**: Addresses must provide a mechanism for error detection when they might be transmitted outside the network.
+3. **Upgradable**: Addresses must be versioned to permit the introduction of new address formats.
+4. **Compact**: Given the above constraints, addresses must be as short as possible.
+
+## Specification
+
+A Filecoin address encoded to a byte array consists of:
+* A **protocol indicator** byte that identifies the type and version of this address.
+* The **payload** used to uniquely identify the actor according to the protocol.
+```
+|----------|---------|
+| protocol | payload |
+|----------|---------|
+|  1 byte  | n bytes |
+```
+
+A Filecoin address encoded to a string consistes of:
+* A **network prefix** character that identifies the network the address belongs to.
+* A **protocol indicator** byte that identifies the type and version of this address.
+* A **payload** used to uniquely identify the actor according to the protocol..
+* A **checksum** used to validet the address. 
```

Comment:
typo on validate, and also maybe note that the checksum isnt always present?

Reply:
I was trying to indicate that the network and checksum won't always be present by have a "Filecoin address encoded to a byte" section and a "Filecoin address encoded to a string". What would make this more clear? Stating that only the bytes encoding will appear on chain?